internal/ciTools/ci.go

```diff
@@ -0,0 +1,33 @@
+package ciTools
+
+import (
+	"log"
+
+	"github.com/kubefirst/kubefirst/configs"
+	"github.com/kubefirst/kubefirst/internal/flagset"
+	"github.com/kubefirst/kubefirst/internal/gitlab"
+	"github.com/kubefirst/kubefirst/internal/repo"
+	"github.com/spf13/viper"
+)
+
```

Comment:
Later to add the github cousin of `DeployGitlab`.. maybe it should be called `DeployOnGitlab`.